models/intel/text-spotting-0001-detector/description/text-spotting-0001-detector.md

```diff
@@ -1,15 +1,15 @@
 # text-spotting-0001-detector
 
-## Use case and High-level description
+## Use Case and High-Level Description
 
-This is text spotting model that means it simultaneously detects and
-recognizes text. The model detects symbol sequences separated by space and does
-recognition without using any dictionary. The model is built on top of Mask-RCNN
+This is a text-spotting model that simultaneously detects and
```

Comment:
> text-spotting

This is analogous to "text detection" or "text recognition", so I don't think there should be a hyphen there.

Reply:
Please fix other instances of this too.